### DISCUSSION TITLE
Atari game display problem

### DISCUSSION THREAD

**Discussion by yupingyuping** [community/community#35503] [Other Feature Feedback, Questions, & Ideas]
I want to display an atari game by env.render("rgb-array").
The evn:" ID = 'BreakoutNoFrameskip-v4' env = gym.make(ID)"
But it doesn't work
my installation version are: python 3.9.12, tensorflow 2.9.1,ale_py0.7.5 keras 2.9.0
In fact I can save suc the pictures in every episode successfully.So I don't think I missed any installation. But I cann't show the graphics like Cartpole,Mountain Car。
Need anyone's help
Many Thanks

Attachment are the picture, later are my codes.
2

import atari_py
from skimage import io
print(atari_py.list_games())

import time
import gym
import matplotlib.pyplot as plt

ID = 'BreakoutNoFrameskip-v4'
env = gym.make(ID)
print("Observation Space: ", env.observation_space)
print("Action Space ", env.action_space)
obs = env.reset()

for i in range(10):
env.render("rgb_array") #(ID, render_mode = "human")
action = env.action_space.sample()
obs, reward, done, info = env.step(action)
jpgname = './pici23-%d.jpg' % i
io.imsave(jpgname, obs)
time.sleep(0.01)
env.close()
